.github/workflows/npm-dev.yaml

```diff
@@ -24,4 +24,6 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           NPM_TAG: "beta"
-          PR_NUMBER_AND_COMMIT_SHA: ${{ github.event.number }}-${{ github.event.pull_request.head.sha }}
+          # Since this only runs on a merge into main, we can't use github.event.number
+          # so we instead use the word "beta" and the PR merge commit SHA
+          PR_NUMBER_AND_COMMIT_SHA: beta-${{ github.sha }}
```

Comment:
I don't _love_ using "beta" where you would expect a PR_NUMBER but this was the least invasive change I could think of. I didn't want to have to add another environment variable and make this more complicated than it needs to be.

Reply:
But then again, we could always change this to `NPM_VERSION` but I don't know if that will make it more complicated. Open to ideas. Just wanted to open a fix quickly.

Reply:
My bad, I reviewed too quickly and missed this I think :stuck_out_tongue: 

In case this is still relevant, I think it is a valid concern and `NPM_VERSION` would describe what this variable is for perfectly.